x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/formatter/TextFormatTests.java

```diff
@@ -265,7 +270,7 @@ private static EsqlQueryResponse regularData() {
                     .appendBytesRef(new BytesRef("Mind Train"))
                     .build(),
                 blockFactory.newIntArrayVector(new int[] { 11 * 60 + 48, 4 * 60 + 40 }, 2).asBlock(),
-                blockFactory.newLongArrayVector(new long[] { GEO.pointAsLong(12, 56), GEO.pointAsLong(-97, 26) }, 2).asBlock(),
+                blockFactory.newBytesRefArrayVector(geoPoints, 2).asBlock(),
```

Comment:
I had left this here for the planned doc-values version. So this test covered both GEO and CARTESIAN, as well as both Long and ByteRef versions. Your simplification here removes the long version. But we can always bring that back when we bring back doc-values.